<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>522</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>735</width>
    <height>522</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>735</width>
    <height>522</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Googri</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_pesquisa">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Pesquise aqui...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pesquisar">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pesquisar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_tipo_pesquisa">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>And</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Or</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>701</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
     </font>
    </property>
    <property name="text">
     <string>Número de páginas indexadas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pageIndex">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Fundo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>481</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>o</string>
    </property>
   </widget>
   <zorder>label_Fundo</zorder>
   <zorder>lineEdit_pesquisa</zorder>
   <zorder>pushButton_pesquisar</zorder>
   <zorder>comboBox_tipo_pesquisa</zorder>
   <zorder>textEdit</zorder>
   <zorder>label</zorder>
   <zorder>label_pageIndex</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>735</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionAbrir"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_pesquisa</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_pesquisar</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>675</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
